static/app/views/issueDetails/streamline/flagSeries.tsx

```diff
@@ -80,6 +48,18 @@ export default function useFlagSeries({query = {}, event}: FlagSeriesProps) {
   } = useOrganizationFlagLog({organization, query});
   const {selection} = usePageFilters();
 
+  const {
+    data: _suspectFlags,
+    isError: _isSuspectFlagsError,
+    isPending: _isSuspectFlagsPending,
+  } = useSuspectFlags({
+    organization,
+    query,
+    firstSeen: group.firstSeen,
+    rawFlagData,
+    event,
+  });
```

Comment:
you don't need to do anything with the return value if its unused
```suggestion
  // Used for analytics
  useSuspectFlags({
    organization,
    query,
    firstSeen: group.firstSeen,
    rawFlagData,
    event,
  });
```